packages/workspace/src/command-line/workspace-generators.ts

```diff
@@ -464,10 +447,8 @@ function createTmpTsConfig(
   return tmpTsConfigPath;
 }
 
-function cleanupTmpTsConfigFile(tmpTsConfigPath) {
-  try {
```

Comment:
I think this `try catch` is intentional. Failures here are okay. The `tmp` file will remain on the system but will be overwritten the next time it is needed.